lib/ranema/actions/add_deprecation_warning_postgresql.rb

```diff
@@ -20,11 +20,13 @@ def perform
       end
 
       def performed?
-        exec_query(
-          "SELECT exists(SELECT * FROM pg_proc WHERE proname = $1)",
-          "SQL",
-          [[nil, trigger_name]]
-        ).to_a.first["exists"]
+        binds = [
+          ActiveRecord::Relation::QueryAttribute.new("trigger_name", trigger_name, ActiveRecord::Type::Value.new)
+        ]
+
+        exec_query("SELECT exists(SELECT * FROM pg_proc WHERE proname = $1)", "SQL", binds)
```

Comment:
Kan volgens mij ook zo, doet volgens mij hetzelfde

```suggestion
        exec_query("SELECT exists(SELECT * FROM pg_proc WHERE proname = $1)", "SQL", [trigger_name])
```

Reply:
Gefixt! Ziet er een hoop rustiger uit zo!